cmd/consensus/main.go

```diff
@@ -346,7 +346,11 @@ func main() {
 			// perform this only if state machine is in initial state
```

Comment:
would suggest documenting the logical block of [lines 341 to 357](https://github.com/onflow/flow-go/blob/caef7c3f7a06f163bb0979dfc05732db7395a48d/cmd/consensus/main.go#L341-L357) as a whole. Something like 

```golang
// store my beacon key for the first epoch post-spork (only if we haven't run this logic before, i.e. state machine is in initial state)
started, err := myBeaconKeyStateMachine.IsDKGStarted(epochCounter)
if err != nil {
  return fmt.Errorf("could not get DKG started flag for root epoch %d: %w", epochCounter, err)
}
if !started {
  epochProtocolState, err := rootSnapshot.EpochProtocolState()
  ⋮

```